src/Drivers/SlackNotificationDriver.php

```diff
@@ -132,7 +134,23 @@ public static function getPrivateChannel() : ?string
      */
     public static function getPublicDriverId(string $notification) : ?string
     {
-        return null;
+        try {
```

Comment:
What will change for another driver?
Can't it be moved to the abstract parent class?

Reply:
i agree, i'll move it.